lib/chef/azure/service.rb

```diff
@@ -117,6 +117,7 @@ def is_running?
   def start_service(bootstrap_directory, log_location)
     puts "#{Time.now} Starting chef-client service ...."
     params = " -c #{bootstrap_directory}\\client.rb -L #{log_location}\\chef-client.log "
+    set_startup_type = %x{powershell.exe -Command "Set-Service -Name 'chef-client' -StartupType automatic"}

```

Comment:
mixlib-shellout library basically for subprocess management and output collection. Here, we didn't collecting any output so that shell_out is not required for this.